doc/ipython-notebooks/converter/HessianLLE.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
```

Comment:
Rather than putting all content into a single cell, split it into multiple ones.
Then dont use source code comments to explain what you are doing, but the notebook markdown in separate cells